registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/resources/bootstrap.properties

```diff
@@ -14,4 +15,16 @@ server.servlet.path=/registrationprocessor/v1/manualverification
 # 5 minutes (should not be done in production)
 health.config.enabled=false
 server.port=8084
-eventbus.port=5720
\ No newline at end of file
+eventbus.port=5720
+
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.show-sql=true
+
+activemq.message.format=text
+
+registration.processor.manual.adjudication.policy.id=mpolicy-default-adjudication

```

Comment:
remove from here, should be read from config server

Reply:
Placed in registration-processor-mz.properties file